crates/xtask/src/history.rs

```diff
@@ -12,7 +12,7 @@ use std::time::Duration;
 const REPO_URL: &str = "https://github.com/open-telemetry/semantic-conventions.git";
 const ARCHIVE_URL: &str =
     "https://github.com/open-telemetry/semantic-conventions/archive/refs/tags/";
-const START_TAG: &str = "v1.22.0";
+const START_TAG: &str = "v1.26.0";
```

Comment:
Semconv [document codegen support with weaver from 1.26.0](https://github.com/open-telemetry/semantic-conventions/blob/e70853397b73ca9cb85b1df45182b6a3e299c85e/docs/non-normative/code-generation.md?plain=1#L87), so this should be fine

Reply:
We discussed this a bit ago.  We should have a principle here.  When weaver was first adopted, there was expectation from some SIGs (Go, I believe) that it should be able to go *backwards* beyond just 1.26.0 when it was first adopted.

I'd like to have some principle here.  We discussed using "first stable semconv" as the version we want to support for at least 2 years (as per support policy in OTEL).

Reply:
I made everything required behind a feature flag and rolled back this change